composer.json

```diff
@@ -17,12 +17,12 @@
     },
     "require": {
         "php": "^7.4|^8.0",
+        "phpunit/phpunit": "^9.5",
```

Comment:
Theoretically, the package now can be used with any testing framework, be it PHPUnit, [atoum](http://atoum.org/) or [behat](https://docs.behat.org/en/latest/). This pull request limits the package to PHPUnit only. That is not particularly bad but we need to decide if the direction is alright.